assets/js/main.js

```diff
@@ -4,15 +4,15 @@ grade_points = {
     "A":8,
     "B+":7,
     "B":6,
-    "C":5,
-    "P":4,
+    "C":5.5,
+    "W":0,
     "F":0,
     "Ab":0,
     "I":0
 };
 
 
-var count = 11;
+var count = 7;

```

Comment:
please keep the count as 6 since there are 6 rows by default. otherwise it causes a bug and adds row 8 instead of 7 when clicking add course button.
<img width="1152" alt="Screenshot 2023-09-02 at 10 08 19 AM" src="https://github.com/optimusam/srmgpa/assets/16255823/c824365f-9ced-4fa9-80a3-72cc57dd6e6c">

Reply:
Done!